### DISCUSSION TITLE
EnduroSat "SpaceDev" and FPrime Integration

### DISCUSSION THREAD

**Discussion by andresmcuevas** [nasa/fprime#2549] [Q&A]
Hello all, I am reaching out to the FPrime community to see if anyone has done any CubeSat flight software development using components sold by the company EnduroSat, specifically in addition to their in-house software solution "SpaceDev."

I am a part of a university CubeSat project that is interested in developing our own flight software using FPrime, and is also interested in purchasing and utilizing EnduroSat subsystems including the On-Board Computer, Electrical Power System, Transceiver, and Antenna. 

From my understanding, these components would ship with a software solution called SpaceDev, that is advertised to streamline the setup of said components. However, this would take away from the experience we hope to gain from developing the flight software in-house, and the overall goal of our project. Therefore, I am hoping to get some feedback on how prohibitive SpaceDev is in terms of FPrime development and integration. Would we be able to utilize SpaceDev alongside our FPrime development, or would we have to forego using SpaceDev altogether. 

Anybody with experience working with SpaceDev would be a great help, but hopefully I could get some insight on how it works with FPrime as well.

**Comment by ebenezerarunkumar** [comment]
@andresmcuevas did you ever end up interfacing the EnduroSat OBC w/ FPrime ? Did you hook up SpaceDev w/ FPrime somehow?
